Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-uncased and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
[INFO] Group mapping used: {'black': 0, 'white': 1}
Parameter 'function'=<function df_map.<locals>.<lambda> at 0x7f8f3b39dbc0> of the transform datasets.arrow_dataset.Dataset._map_single couldn't be hashed properly, a random hash was used instead. Make sure your transforms and parameters are serializable with pickle or dill for the dataset fingerprinting and caching to work. If you reuse this transform, the caching mechanism will consider it to be different from the previous calls and recompute everything. This warning is only showed once. Subsequent hashing failures won't be showed.
Map: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████| 10000/10000 [00:02<00:00, 4328.50 examples/s]
[INFO] Group mapping used: {0: 0, 1: 1}
Map: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100/100 [00:00<00:00, 3639.15 examples/s]
{'train_runtime': 0.5549, 'train_samples_per_second': 180.204, 'train_steps_per_second': 3.604, 'train_loss': 0.0656653493642807, 'epoch': 1.0}

=== Iteration 1 ===
[INFO] Group mapping used: {0: 0, 1: 1}
Map: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100/100 [00:00<00:00, 3088.54 examples/s]
Casting the dataset: 100%|███████████████████████████████████████████████████████████████████████████████████████████| 10000/10000 [00:00<00:00, 507404.13 examples/s]
Lambda: 0.0000, Violation: -0.4992
[Epoch 1, Batch 1/157] Loss: -0.0846, Violation: -0.4992
Lambda: 0.0000, Violation: -0.4977
[Epoch 1, Batch 157/157] Loss: nan, Violation: -0.4977
Training done for C-ERM
Start Calculation of AUC on whole D -> might take a while
Done with calculation whole D
Casting the dataset: 100%|███████████████████████████████████████████████████████████████████████████████████████████| 10000/10000 [00:00<00:00, 420477.39 examples/s]
Lambda: 0.0000, Violation: -0.4973
[Epoch 1, Batch 1/157] Loss: 0.0113, Violation: -0.4973
Lambda: 0.0000, Violation: -0.4988
[Epoch 1, Batch 157/157] Loss: nan, Violation: -0.4988
Training done for C-ERM
Start Calculation of AUC on whole D -> might take a while
Done with calculation whole D
Stopping: ΔAUC(h1-h2) = 0.0017 within 2ε tolerance
Traceback (most recent call last):
  File "/mnt/sda/david/activeAuditing/new_activeAuditing/AcitveAuditing_140725_refactored/main.py", line 55, in <module>
    main()
  File "/mnt/sda/david/activeAuditing/new_activeAuditing/AcitveAuditing_140725_refactored/main.py", line 51, in main
    runner.run()
  File "/mnt/sda/david/activeAuditing/new_activeAuditing/AcitveAuditing_140725_refactored/audit_run.py", line 142, in run
    T = self.refine_until_converged(surrogate, tokenizer, base_model, inputs_D, df_D_mapped, S)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/sda/david/activeAuditing/new_activeAuditing/AcitveAuditing_140725_refactored/audit_run.py", line 116, in refine_until_converged
    return new_points
           ^^^^^^^^^^
UnboundLocalError: cannot access local variable 'new_points' where it is not associated with a value
